neurobooth_os/iout/flir_cam.py

```diff
@@ -190,44 +189,45 @@ def prepare(self, name="temp_video"):
         with meta.get_database_connection() as db_conn:
             meta.post_message(Request(source='Flir', destination='CTR', body=msg_body), conn=db_conn)
         self.streaming = True
+        self.manifest_dict["frame_rate"] = self.FRAME_RATE_OUT
+        self.manifest_dict['frame_height'] = im.shape[0]
+        self.manifest_dict['frame_width'] = im.shape[1]
+        self.manifest_dict['frame_depth'] = im.shape[2]
 
     def record(self):
         self.logger.debug('FLIR: LSL Thread Started')
         self.recording = True
-        self.frame_counter = 0
-        self.save_thread = threading.Thread(target=self.camCaptureVid)
```

Comment:
The camCaptureVid function is not used anymore at all. IMHO keep it in the script for now but comment it out. If the tests go fine and we decide to keep current changes in production, then we could remove it completely.

Reply:
I just deleted that function.  if we have to revert we can go back to the prior release.  Leaving commented out code isn't a good practice.